nextjs-wordles/utils/variables.js

```diff
@@ -0,0 +1,96 @@
+export const ALLOWED_LETTERS = [
+    'q',
+    'w',
+    'e',
+    'r',
+    't',
+    'y',
+    'u',
+    'i',
+    'o',
+    'p',
+    'a',
+    's',
+    'd',
+    'f',
+    'g',
+    'h',
+    'j',
```

Comment:
Może spróbuj użyć jednego rodzaju liter absolutnie wszędzie. Klawiatura, stan, tablica itp. [Kapitalizacja na tablicy może być zrobiona za pomocą CSS](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform). To zwiększy pewność podczas wprowadzania później zmian.